packages/gestalt/src/Button.js

```diff
@@ -1,6 +1,5 @@
 // @flow strict
 import type { Node } from 'react';
-
 import { forwardRef, useImperativeHandle, useRef } from 'react';
```

Comment:
If you wanted to consolidate the `type Node` import with the rest of the `'react'` imports, I wouldn't complain 😅

Reply:
DONE